frontend/src/components/Plans.tsx

```diff
@@ -62,7 +63,7 @@ export const Plans: React.FC = () => {
     purchasing: state.plans.purchasing,
     license: selectOwnRemoteitLicense(state),
   }))
-  const getDefaults = () => {
+  function getDefaults(): IPurchase {
```

Comment:
why are you taking out const ?

Reply:
I think I just changed it so that I knew how to specify the return value in typescript